test/TestServerProjects/body-complex/Generated/Models/MyBaseType.cs

```diff
@@ -16,7 +16,6 @@ public MyBaseType()
         /// <summary> The constant value Kind1. </summary>
         public string Kind { get; internal set; } = "Kind1";
         public string PropB1 { get; set; }
-        /// <summary> MISSING·SCHEMA-DESCRIPTION-OBJECTSCHEMA. </summary>
-        public MyBaseHelperType Helper { get; set; }
+        public string PropBH1 { get; set; }
```

Comment:
Didn't we ever use this in test?

Reply:
It isn't used in any test server tests. It is defined in the swagger for it, but no tests set/get that value.